docs/HowToArchiveJournal.md

```diff
@@ -0,0 +1,31 @@
+# NuoDB Storage Manager (SM) Journal in Kubernetes
+
+## Introduction
+
+The storage manager, SM for short, is responsible for maintaining a complete copy of the database.
+The atoms, database elements, are stored to either local disk, or in a separate volume.
+These atoms are written by a specific module called the `archive`.
+However, the archive doesn’t have any special mechanisms to satisfy the durability requirements.
+In addition to the archive, NuoDB storage managers also maintain a write-ahead-log called the `journal`.
+The journal has the onerous task of ensuring durability in the face of an unexpected process termination.
+E.g., a power loss, machine meltdown, or an unexpected cloud outage.
+The journal will make sure that the archive is reconstructed in a consistent state when coming back online.
+
+For more info on NuoDB database journaling, please consult the official [NuoDB docs](https://doc.nuodb.com/nuodb/latest/database-administration/about-database-journaling/).
+
+## Separating the Journal from the Archive
+
+Since the `journal` has to write all commits durably to disk, the speed of the disk directly influences to the commit latency.
+To achieve the best performance, NuoDB recommends placing the `journal` on the fastest disk available.
+
+On the other hand, the `archive` can be located on slightly slower disks since writes to the archive are asynchronous.
+
+By default, the `journal` is located in a subdirectory of the `archive`.
+To achieve the best cost vs speed tradeoff, you can separate the journal from the archive.
```

Comment:
This implies we would _only_ put the journal on fast disk.  In reality we would probably always want (in production) both archive _and_ journal on their own fast _and_ separate disks.

If the archive is on slower disk then we improve "burst" capacity but at some point will throttle.  I think its something like this from worst to best:
1. archive + journal on same slow disk
2. archive + journal on separate slow disk
3. archive + journal on on same fast disk
4. archive on slow disk, journal on fast disk (good burst capacity)
5. archive + journal on separate fast disk

Reply:
I will open a separate PR to wrap up the docs here!

Reply:
I don't know if cloud providers actually make a distinction between latency and throughput, but the system would actually be bound by the write latency of the journal disk and the write throughput of the archive disk.

It would be possible to schedule writes to the archive in a way that maximizes write throughput so that a slower archive disk can keep up with a faster journal disk. Even though there is a lot of room for improvement, I think we probably already benefit from write batching done by the filesystem since writes to the archive are asynchronous and we are not fence-posted with lots of `fsync` calls, the way writes to the journal are. And this would probably be more effective if the journal has its own disk, since journal `fsync`s would not interfere with batching of archive writes.

Reply:
I don't remember seeing the ability in any clouds to _choose_ your latency.  I suspect it will naturally improve with higher / provisioned IOPS because of the backing disk being SSD based.  AWS I believe is supposed to deliver "single digit latency" on SSD based volumes.  At least having separate  PV's would also allow the use of local SSD volumes to get sub ms performance of the journal.  There is probably a task here (now we have the ability to separate) to find the best combination - for example there are also throughput optimised disk types.